### DISCUSSION TITLE
Image doesnt appear

### DISCUSSION THREAD

**Discussion by fbrliatr** [filamentphp/filament#13706] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hi, can someone help me to solve how is my image doesnt appear? it saved in database and in folder storage/app/public/posts. I've done the php artisan storage:link either. so why the image doesnt show in my web? Thank you. I might drop the output, coding, and the folder image below

![Screenshot 2024-07-26 022721](https://github.com/user-attachments/assets/54f8a2a0-f32b-448c-b26b-539ac80d174d)
![image](https://github.com/user-attachments/assets/b7c62f73-d2d3-4603-9daf-5dca8b3aae4e)
![image](https://github.com/user-attachments/assets/ca5b430f-5271-4b94-a8ff-99ed10189802)

**Comment by awcodes** [comment]
Storage::get('posts/image.png')->url()

**Reply by fbrliatr** [reply]
thank youu! the logo has been successfull appeared. but I have another question (I dont know if this would be the same answer or not but i will ask just in case). So, I have a form with FileUpload 'image' but the image doesnt want to appear. my storage link is done and the link is going to **public/storage** BUT the image is stored to **storage/app/public**. I dont get what's wrong. Can you help me once again?

Create Data Code:
```
Forms\Components\FileUpload::make('image')
                ->label('Upload Gambar')
                ->required()
                ->image()
                ->columnSpan('full'),
                // ->square(),
```
Get Data Code
```
Tables\Columns\ImageColumn::make('image')
                    ->label('Gambar')
                    ->searchable()
                    ->sortable(),
```

**Reply by leandrocfe** [reply]
Try to add `->disk('public')` in the FileUpload

**Reply by fbrliatr** [reply]
still doesnt work
